packages/contracts-bedrock/scripts/deploy/DeployOPChain.s.sol

```diff
@@ -81,53 +143,133 @@ contract DeployOPChain is Script {
             disputeMaxClockDuration: _input.disputeMaxClockDuration,
             useCustomGasToken: _input.useCustomGasToken
         });
+    }
 
-        vm.broadcast(msg.sender);
-        IOPContractsManager.DeployOutput memory deployOutput = opcm.deploy(deployInput);
-
-        vm.label(address(deployOutput.opChainProxyAdmin), "opChainProxyAdmin");
-        vm.label(address(deployOutput.addressManager), "addressManager");
-        vm.label(address(deployOutput.l1ERC721BridgeProxy), "l1ERC721BridgeProxy");
-        vm.label(address(deployOutput.systemConfigProxy), "systemConfigProxy");
-        vm.label(address(deployOutput.optimismMintableERC20FactoryProxy), "optimismMintableERC20FactoryProxy");
-        vm.label(address(deployOutput.l1StandardBridgeProxy), "l1StandardBridgeProxy");
-        vm.label(address(deployOutput.l1CrossDomainMessengerProxy), "l1CrossDomainMessengerProxy");
-        vm.label(address(deployOutput.optimismPortalProxy), "optimismPortalProxy");
-        vm.label(address(deployOutput.ethLockboxProxy), "ethLockboxProxy");
-        vm.label(address(deployOutput.disputeGameFactoryProxy), "disputeGameFactoryProxy");
-        vm.label(address(deployOutput.anchorStateRegistryProxy), "anchorStateRegistryProxy");
-        vm.label(address(deployOutput.permissionedDisputeGame), "permissionedDisputeGame");
-        vm.label(address(deployOutput.delayedWETHPermissionedGameProxy), "delayedWETHPermissionedGameProxy");
-        // TODO: Eventually switch from Permissioned to Permissionless.
-        // vm.label(address(deployOutput.faultDisputeGame), "faultDisputeGame");
-        // vm.label(address(deployOutput.delayedWETHPermissionlessGameProxy), "delayedWETHPermissionlessGameProxy");
+    /// @notice Converts Types.DeployOPChainInput to IOPContractsManagerV2.FullConfig.
+    /// @param _input The input parameters.
+    /// @return config_ The deployed input parameters.
+    function toOPCMV2DeployInput(Types.DeployOPChainInput memory _input)
```

Comment:
interesting that #18079 didn't require this. It looks like opcmv2.deploy() is not tested yet.